src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -748,9 +748,25 @@ namespace ngraph
                 template <>
                 void CPUAssignment::ASSIGN_DECL(ngraph::op::QuantizedConvolution)
                 {
+                    auto qconv = static_cast<ngraph::op::QuantizedConvolution*>(node);
+                    auto input_zero_point =
+                        dynamic_pointer_cast<ngraph::op::Constant>(qconv->get_argument(3));
+                    auto filter_zero_point =
+                        dynamic_pointer_cast<ngraph::op::Constant>(qconv->get_argument(5));
+                    auto output_zero_point =
+                        dynamic_pointer_cast<ngraph::op::Constant>(qconv->get_argument(7));
                     if (node->get_input_element_type(0) == element::u8 &&
                         node->get_input_element_type(1) == element::i8)
                     {
+                        // Mkldnn assumes zero point to be zero
+                        if (input_zero_point == nullptr || filter_zero_point == nullptr ||
+                            output_zero_point == nullptr || !(ngraph::is_zero(input_zero_point)) ||
+                            !(ngraph::is_zero(filter_zero_point)) ||
+                            !(ngraph::is_zero(output_zero_point)))
```

Comment:
is_zero checks for node type being a constant. you dont need to do the type check here.

Reply:
is_zero checks if a constant node is zero or not. I do dynamically cast it to Constant and check for if its value to be zero. Zero point should be zero and Constant for mkldnn kernel to kick in